backend/src/database/seeds/01_users.js

```diff
@@ -2,30 +2,36 @@
 exports.seed = function(knex) {
   const tableName = "users";
   // Deletes ALL existing entries
-  return knex(tableName).del()
+  return knex.raw('SET session_replication_role = \'replica\';')
+    .then(function () { return knex(tableName).del() })
```

Comment:
```suggestion
    .then(() => knex(tableName).del())
```

Reply:
thanks for pointing this out. i made these changes for #44